vcluster/configure/vcluster-yaml/external/platform/api-key.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: apiKey
+sidebar_label: apiKey
+---
+
+import ApiKey from '@site/vcluster/_partials/config/external/platform/apiKey.mdx'
+import External from '@site/vcluster/_partials/config/external.mdx'
+
+<External />
+
+## Config reference
+
+<ApiKey />
+
+## Permissions
+You need to add the `spec.scope.roles[*].role=vcluster` in the Platform Access Key OwnedAccessKey CRD. The UI does not support this other than allowing you to enter freeform in the CRD editor.
+
+```yaml
+spec:
+  scope:
+    roles:
+      - role: "vcluster"
+```
+
+## Secret
+
+:::note
```

Comment:
This can be removed as we never really published the yaml docs and external was only introduced recently.